styleguide/source/_patterns/03-organisms/dr-finder-search/dr-finder-search.twig

```diff
@@ -0,0 +1,5 @@
+{# @copyright Copyright (c) 2022 Palantir.net #}
+<div class="dr-finder-search">
+{# Include the dr finder search fields #}
+    <h{{ drFinderSearch.heading.level}} class="{{ drFinderSearch.heading.class }}">{{ drFinderSearch.heading.text|raw }}</h{{ drFinderSearch.heading.level }}>
```

Comment:
The spacing on the `{{` and `}}` elements is inconsistent.

Reply:
got it. Thanks!